builds/android/src/org/easyrpg/player/game_browser/GameBrowserActivity.java

```diff
@@ -122,6 +129,8 @@ public void displayGameList(){
 			adapter = new GameListAdapter(this, project_list);
 		}
 		
+		Log.i("Browser", error_list.size() + " jeux trouvés");
```

Comment:
Yep no problem ! This PR is kind of a WIP (to discuss about how should the system should be), the final version will be cleaned of logs ^^. Don't hesitate to interact with further pushes if you think the system should be different :).